pg_log_userqueries.c

```diff
@@ -759,10 +931,15 @@ pgluq_log(const char *query)
 	else
 		logged_in_utility_hook = false;
 
-	/* when log regexp statement is set do not log the query if it doesn't match the regexp */
-	if ((log_query != NULL) && (regexec(&query_regexv, query, 0, 0, 0) != 0))
-		return;
+	/* Filter querries according to the white and balck list*/
+	if (log_query != NULL || log_query_blacklist != NULL) {
+		if ( ! pgluq_checkitem(	appname,
```

Comment:
This doesn't compile. I get a:

```
pg_log_userqueries.c:936:27: error: ‘appname’ undeclared (first use in this function)
  936 |   if ( ! pgluq_checkitem( appname,
      |                           ^~~~~~~
```

Changing it to application_name allows the compilation to finish, but I'm not sure this is a real fix.

Reply:
Fixed